node/pkg/driver/node_utils.go

```diff
@@ -580,6 +597,27 @@ func (d NodeUtils) GetBlockVolumeStats(mpathDevice string) (VolumeStatistics, er
 	return volumeStats, nil
 }
 
+func (d NodeUtils) IsVolumePathMatchesVolumeId(volumeId string, volumePath string) (bool, error) {
+	volumeUuid := d.GetVolumeUuid(volumeId)
+	volumeIdByVolumePath, err := d.osDeviceConnectivityHelper.GetVolumeIdByVolumePath(volumePath, volumeUuid)
```

Comment:
why does it need the volume (uu)id?
it's not clear to me from the name (or at all)

Reply:
yes you right, I tried now to change it and do it clearer, what do you think?

Reply:
still not sure what `volumeIdByVolumePath` means. is it `volumeIdOfVolumePath`?

Reply:
I removed this function, but yes `volumeIdOfVolumePath` is what it means

Reply:
done